src/signals/incident/definitions/wizard-step-2-vulaan/wonen.js

```diff
@@ -0,0 +1,864 @@
+import { Validators } from 'react-reactive-form';
+
+import FormComponents from '../../components/form';
+import IncidentNavigation from '../../components/IncidentNavigation';
+
+export default {
+  controls: {
+    custom_text: {
+      meta: {
+        label: 'Dit hebt u net ingevuld:',
+        type: 'citation',
+        value: '{incident.description}',
+        ignoreVisibility: true,
+      },
+      render: FormComponents.PlainText,
+    },
+
+    // woningdelen
+    extra_wonen_woningdelen_vermoeden: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wat zich in deze woning afspeelt?',
+        subtitle: 'Vermoedens over bijvoorbeeld illegale activiteiten',
+        shortLabel: 'Vermoeden',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      render: FormComponents.TextInput,
+    },
+    extra_wonen_woningdelen_eigenaar: {
+      meta: {
+        ifOneOf: {
+          subcategory: [
+            'woningdelen',
+          ],
+        },
+        label: 'Weet u wie de eigenaar is van de woning?',
+        shortLabel: 'Naam Eigenaar',
+        pathMerge: 'extra_properties',
+        className: 'col-sm-12 col-md-8',
+      },
+      options: {
+        validators: [Validators.required],
```

Comment:
Should this be required? If so, shouldn't it be a set of radio buttons?

Reply:
Herjen would like to keep this as it is.